### PR TITLE
CLDR-17566 converting getting started p1

### PR DESCRIPTION
**Pull request by chpy04**
[CLDR-17566](https://unicode-org.atlassian.net/browse/CLDR-17566)

Part of the [CLDR --> GFM conversion project](https://docs.google.com/document/d/1NoQX0zqSYqU4CUuNijTWKQaphE4SCuHl6Bej2C4mb58/edit)

Converting: https://cldr.unicode.org/translation/getting-started,
https://cldr.unicode.org/translation/getting-started/guide,
https://cldr.unicode.org/translation/getting-started/survey-tool-phases,
https://cldr.unicode.org/translation/getting-started/vetting-view,
https://cldr.unicode.org/translation/getting-started/errors-and-warnings

Text diffs can be seen in diffs under the second commit in this PR.
Use the 📄 button to see a rich view, for comparison.

ALLOW_MANY_COMMITS=true

[CLDR-17566]: https://unicode-org.atlassian.net/browse/CLDR-17566?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/site/TEMP-TEXT-FILES/errors-and-warnings.txt [hunk 1]
```diff
@@ -3,26 +3,27 @@ Handling Errors and Warnings
...
 Currency Symbols & Names
 The characters ‎[…]‎ should not be used (Must fix)
 For what to do, see Characters, in the section Handling Warnings. While these are categorized as warnings, every effort should be made to fix them.
 Unquoted special character '.' in pattern (Must fix)
 Number patterns can only contain an unquoted . when it is the decimal separator.
-In other cases, such as to finish an abbreviation, it must be quoted, as in:    000 tūkst'.'
+In other cases, such as to finish an abbreviation, it must be quoted, as in: 000 tūkst'.'
 For more information, see Number Patterns.
 Errors in Others column can be ignored. For example, this is an example of an Inheritance value in the Others column where the Language (Country/region) names have a collision issue. Do not vote on values that have errors.
+alt-text
```
Comment (macchiati) [inline docs/site/TEMP-TEXT-FILES/errors-and-warnings.txt]: The introduction of "alt-text" looks like an error.

File: docs/site/TEMP-TEXT-FILES/guide.txt [hunk 2]
```diff
@@ -1,31 +1,34 @@
...
-Import of old votes is automatically handled when you log in for all your votes matching the latest released data. If you have voted previously, upon first log-in, you will see a message showing the number of your votes that matched the currently winning votes that have been auto-imported. 
-Import of your old votes will take some time, but this will be done at 1st log-in only.
-Import old votes manually. You can still import your old voted data that DO NOT match the last released data. 
-Go to the  ☰ Menu, look under My Votes, and choose Import Old Votes.  
+Go to http://st.unicode.org/cldr-apps/survey/ and log in. 
+Import of old votes is automatically handled when you log in for all your votes matching the latest released data. If you have voted previously, upon first log-in, you will see a message showing the number of your votes that matched the currently winning votes that have been auto-imported.
+Import of your old votes will take some time, but this will be done at 1st log-in only. 
+Import old votes manually. You can still import your old voted data that DO NOT match the last released data.
+Go to the G☰ Menu, look under My Votes, and choose Import Old V****otes.
```
Comment (macchiati) [inline docs/site/TEMP-TEXT-FILES/guide.txt]: This also looks like an error.

File: docs/site/TEMP-TEXT-FILES/guide.txt [hunk 3]
```diff
@@ -107,43 +118,52 @@ Code: this links to the field in survey tool. Click on it to go to the item in t
 English: The English value is highlighted in blue.
 Winning XX: The currently winning value is highlighted in green.
 Hide checkbox: For items that can be hidden a checkbox to hide that option appears on the far right.
+
 How to handle different categories
 Following are guidelines on best practices for handling items under each category in Dashboard.
 Missing
-These are items where there is no localization provided by any contributor. Click on the line to be taken to the item in the Survey Tool where items are highlighted and you can add a translation. When you fix a Missing item it will turn to Changed.
+The are items where there is no localization provided by any contributor. Click on the line to be taken to the item in the Survey Tool where items are highlighted and you can add a translation. When you fix a Missing item it will turn to Changed.
```
Comment (macchiati) [inline docs/site/TEMP-TEXT-FILES/guide.txt]: Also looks like an error

**Review by macchiati — CHANGES_REQUESTED** [review]
Just a few items to fix. Please let me kmow in slack when you have addressed those.

File: docs/site/TEMP-TEXT-FILES/errors-and-warnings.txt (hunk 1 above)
Comment (chpy04) [reply docs/site/TEMP-TEXT-FILES/errors-and-warnings.txt]: alt-text is just what shows up in the text file for any images when I copy directly from the output of markdown. I try to clean them up for the most part but they aren't in the actual md files.

Comment (macchiati) [reply docs/site/TEMP-TEXT-FILES/errors-and-warnings.txt]: got it

File: docs/site/TEMP-TEXT-FILES/guide.txt (hunk 2 above)
Comment (macchiati) [reply docs/site/TEMP-TEXT-FILES/guide.txt]: Still a problem. There is no 
> Go to the **G☰ Menu**

There is no "G☰" menu. You need to remove the "G". Here is what it looks like; in this image, the symbol after "Vetting"

Screenshot 2024-06-19 at 16 43 23

https://st.unicode.org/cldr-apps/v#/de//

File: docs/site/TEMP-TEXT-FILES/guide.txt (hunk 3 above)
Comment (macchiati) [reply docs/site/TEMP-TEXT-FILES/guide.txt]: Looks like you fixed this one.

**Review by macchiati — CHANGES_REQUESTED** [review]
One small item left.

**Review by macchiati — APPROVED** [review]

**Review by srl295 — APPROVED** [review]

**Pull request merged**